docs(root): add better highlights

evenstensberg opened the pull request:
## Summary of the changes
Makes the README a bit nicer.

## Related issue
N/A

## Checklist

### General 

- [x] Changes to docs package checked and committed.
- [x] All acceptance criteria reviewed and met. 

### Testing

- [x] Relevant unit tests and visual regression tests added. 
- [x] Visual testing against Figma component specification completed. 
- [x] Playground stories in React Storybook up to date, with any prop changes and additions addressed. 

### Accessibility 

- [x] Accessibility Insights FastPass performed.
- [x] A11y unit test added and yields no issues.
- [x] A11y plug-in on Storybook yields no issues. 
- [x] Manual screen reader testing performed using NVDA and VoiceOver. 
- [x] Manual keyboard testing for keyboard controls and logical focus order. 
- [x] Correct roles used and ARIA attributes used correctly where required. 
- [x] Logical heading structure is maintained, and the HTML elements used for headings can be changed to fit within the wider page structure. 

### Resize/zoom behaviour 

- [x] Page can be zoomed to 400% with no loss of content. 
- [x] Screen magnifier used with no issues. 
- [x] Text resized to 200% with no loss of content.
- [x] Text spacing increased as per the [WCAG 1.4.12 success criterion](https://www.w3.org/TR/WCAG21/#text-spacing) with no loss of content.

### System modes

- [x] Browser setting 'prefers reduced motion' tested. No animations or motion visible whilst this setting is on. 
- [x] Windows High Contrast mode tested with no loss of content.
- [x] System light and dark mode tested with no loss of content.
- [x] Browser support tested (Chrome, Safari, Firefox and Edge). 

### Testing content extremes

- [x] Min/max content examples tested with no loss of content or overflow. 
- [x] All prop combinations work without issue. 
- [x] Tested for FOUC (Flash of Unstyled Content) in both SSR (Server-Side Rendering) and SSG (Static Site Generation) settings.
- [x] Controlled and uncontrolled input components tested.

GCHQ-Developer-112 commented on README.md:
```diff
@@ -118,4 +121,4 @@ Visit our websites to learn more about:
 
-© Crown copyright 2022
\ No newline at end of file
+© Crown copyright 2024
```
Please revert this back to 2022

GCHQ-Developer-112 reviewed (changes requested):
Changes look good overall.  
Left a comment on the README.md
Please could you also make the commit message a bit more reflective of the change made
Thank you for your contribution!

evenstensberg commented:
@GCHQ-Developer-112 I've fixed the commit message and added a more detailed explaination in the commit body. PTAL (Please Take A Look).

I have a question: Do I need to explain thoroughly in the commit body? I know some people enforce it, but I haven't worked on so many projects that require you to explain in detail if the commit message itself is explanatory.

GCHQ-Developer-112 commented:
> @GCHQ-Developer-112 I've fixed the commit message and added a more detailed explaination in the commit body. PTAL (Please Take A Look).
> 
> I have a question: Do I need to explain thoroughly in the commit body? I know some people enforce it, but I haven't worked on so many projects that require you to explain in detail if the commit message itself is explanatory.

The short commit summary is more important than the longer description. The short summary is what shows up in the CHANGELOG.md. In the past, we've had feedback that the commit history in the CHANGELOG.md is not the clearest so that's something we bear in mind.
